tests/platform_tests/api/test_sfp.py

```diff
@@ -262,7 +262,7 @@ def is_xcvr_optical(self, xcvr_info_dict):
             spec_compliance_dict = ast.literal_eval(xcvr_info_dict["specification_compliance"])
             if xcvr_info_dict["type_abbrv_name"] == "SFP":
                 compliance_code = spec_compliance_dict.get("SFP+CableTechnology")
-                if compliance_code == "Passive Cable":
+                if compliance_code == "Passive Cable" or compliance_code == "Unknown":
```

Comment:
@andywongarista what if an Active Cable also specify the compliance code incorrectly? Do we need to support such rouge SFPs? Can we replace such rouge SFPs with a valid ones rather than workaround the test